style.css

```diff
@@ -0,0 +1,228 @@
+body {
+    height: 100vh;
+    width: 100vw;
+    background-color: darkred;
+    margin: 0;
+}
+
+.header1 {
+    background-color: black;
+    height: 20vh;
```

Comment:
If you do vh/vw/vmin on the outside elements, you can use percentages on the inside stuff